packages/config/src/defaultTestConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { PrismaConfig } from './PrismaConfig'
```

Comment:
```suggestion
import type { PrismaConfig } from './PrismaConfig'
```

Reply:
Addressed in https://github.com/prisma/prisma/pull/26303/files already